lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -92,6 +92,14 @@ public static function get($ids) {
 
     $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids]);
 
+    if (!user_access('administer modules') && !$results) {
+      $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids], TRUE);
```

Comment:
I don't think it can because if the user doesn't have permission to administer modules, the `flagged` param will get ignored. This is set [here](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc#L133) so that's why I used the param to change that.